configs/jester.cfg

```diff
@@ -0,0 +1,22 @@
+# input loading config
+# Not relevant for local CSV.
```

Comment:
If some parameters are not used is it possible to remove them from this csv (e.g., s3_bucket)?

Reply:
It throws an error when I try to remove those parameters, but I've changed the config file to move those parameters to the top with a comment explaining that they aren't used.